src/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+import { createStore } from 'redux';
+import { devToolsEnhancer } from 'redux-devtools-extension';
+import reducer from './books';
+
```

Comment:
According to the project requirements, we should use [combinereducers](https://redux.js.org/usage/structuring-reducers/using-combinereducers
)
![image](https://user-images.githubusercontent.com/31897434/128092747-cc27ab38-2bd4-49ad-b281-652cf9befa84.png)

Reply:
I don't think is necessary to use a combine reducer since there is only one reducer and it's for books